src/mapgen/cavegen.cpp

```diff
@@ -74,6 +74,8 @@ void CavesNoiseIntersection::generateCaves(MMVManip *vm,
 	assert(vm);
 	assert(biomemap);
 
+	PseudoRandom ps(86219);
```

Comment:
the method should receive a PseudoRandom (like `CavesRandomWalk::makeCave`) so it can be made to depend on the blockseed